pulp_2to3_migrate/app/plugin/content.py

```diff
@@ -147,8 +147,8 @@ def __init__(self, model):
             batch_size = math.ceil(total_pulp2content / max_coro)
         batch_count = math.ceil(total_pulp2content / batch_size)
 
-        with ProgressBar(message='Migrating {} content to Pulp 3'.format(content_type.upper()),
-                         total=total_pulp2content) as pb:
+        with ProgressReport(message='Migrating {} content to Pulp 3'.format(content_type.upper()),
+                            code='migrating.content', total=total_pulp2content) as pb:
```

Comment:
@fabricio-aguiar ,
Could you please include content type into the code since we have `migrating.content` code already?
e.g. `code='migrating.{}.content'.format(content_type)` or whichever way you prefer to do string formatting.